src/index.ts

```diff
@@ -20,7 +20,6 @@ import {
   getNodeArch,
   ensureIsTruthyString,
   isOfficialLinuxIA32Download,
-  withTempDirectory,
 } from './utils';
```

Comment:
'withTempDirectory' is defined but never used.eslint[@typescript-eslint/no-unused-vars](https://github.com/typescript-eslint/typescript-eslint/blob/v2.34.0/packages/eslint-plugin/docs/rules/no-unused-vars.md)